src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random rand = new Random();
+
+    public Random getRand() {
+        return rand;
+    }
+
+    public void setRand(Random rand) {
+        this.rand = rand;
+    }
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), rand.nextInt(100));
```

Comment:
Magic numbers should be avoided; consider defining a constant for the number 100 to give it meaning within the context of this method.